helm/qontract-reconcile/templates/template.yaml

```diff
@@ -132,6 +132,16 @@ objects:
           - name: APP_INTERFACE_STATE_BUCKET_ACCOUNT
             value: "${APP_INTERFACE_STATE_BUCKET_ACCOUNT}"
           {{- end }}
+          - name: UNLEASH_API_URL
+            valueFrom:
+              secretKeyRef:
+                name: unleash
+                key: API_URL
+          - name: UNLEASH_CLIENT_ACCESS_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: unleash
```

Comment:
`app-interface-token` (or `app-interface-unleash-token`, if you changed it there)

Reply:
answered in https://gitlab.cee.redhat.com/service/app-interface/merge_requests/4816#note_1298642